ansible_collections/arista/avd/plugins/filter/convert_dicts.py

```diff
@@ -8,32 +8,31 @@
 import os
 
 
-def convert_dicts(dictionary, primary_key="name", secondary_key="items"):
+def convert_dicts(dictionary, primary_key="name", secondary_key=None):
     """
     The `arista.avd.convert_dicts` filter will convert a dictionary containing nested dictionaries to a list of
-    dictionaries It inserts the outer dictionary keys into each list item using the primary_key `name` (key name is
-    configurable) and if there is a list of dictionary in the dictionary value,it inserts this value list to
-    secondary key "items" (key name is configurable).
+    dictionaries.It inserts the outer dictionary keys into each list item using the primary_key `name` (key name is
```

Comment:
same comment as above - need to add back a space before `It`

Reply:
fixed